app/imageswap-init/imageswap-init.py

```diff
@@ -1112,12 +1154,19 @@ def init_mwc(namespace, imageswap_tls_byoc):
             sys.exit(1)
 
     configuration = client.Configuration()
-    core_api = client.CoreV1Api(client.ApiClient(configuration))
-    admission_api = client.AdmissionregistrationV1beta1Api(client.ApiClient(configuration))
+    configuration.ssl_ca_cert = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+
+    core_api = client.CoreV1Api(client.ApiClient())
+    admission_api = client.AdmissionregistrationV1Api(client.ApiClient())
```

Comment:
I think this can be done using a native python client config method like this:

```python
configuration = client.Configuration().get_default_copy()
core_api = client.CoreV1Api(client.ApiClient(configuration))
admission_api = client.AdmissionregistrationV1Api(client.ApiClient(configuration))
```

the `.get_default_copy()` method will automatically read the ca cert from the in-cluster config

Reply:
I'll try it.